arkouda/numeric.py

```diff
@@ -30,7 +30,8 @@ def abs(pda : pdarray) -> pdarray:
         repMsg = generic_msg("efunc {} {}".format("abs", pda.name))
         return create_pdarray(repMsg)
     else:
-        raise TypeError("must be pdarray {}".format(pda))
+        raise TypeError("must be pdarray {}, not {}".\
+                                   format(pda.__class__.__name__))
```

Comment:
should there be two arguments to this `format` ?

Reply:
@mhmerrill argh, good call! yes, indeed, will fix